sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -38,6 +37,26 @@
  */
 public class SensitivityAnalysisResult {
 
+    /*
+     * Key type for sensitivity value when stored by contingency, function and variable id
+     */
+    private class SensitivityValueKey extends Triple<String, String, String> {
```

Comment:
I was thinking about fully defining this object not inherit from Triple.
So that could can also remove the enummap